symbols/amenity/parcel_locker.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I strongly oppose making it a requirement for style developers to always hand edit plain SVGs saved from inkscape to be acceptable for this style.  This would mean a significant additional hurdle for designers to contribute here - and that for no good reason.  If we want to not have such code in the SVGs (though i see no reason why this is actually a substantial problem) we should provide a script that does such sanitation.

Reply:
> we should provide a script that does such sanitation

Good idea